File: Examples/CSharp/Pages/GetNumberOfPages.cs
using System.IO;
using Aspose.Note;
using System;

namespace Aspose.Note.Examples.CSharp.Pages
{
    using System.Linq;

    public class GetNumberOfPages
    {
        public static void Run()
        {
            // ExStart:GetNumberOfPages
            // ExFor:Document
            // ExSummary:Shows how to get page's count of a document.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Pages();

            // Load the document into Aspose.Note.
            Document oneFile = new Document(dataDir + "Aspose.one");

            // Get number of pages
            int count = oneFile.Count();

            // Print count on the output screen
            Console.WriteLine(count);
            
            // ExEnd:GetNumberOfPages
        }
    }
}

File: Examples/VisualBasic/Pages/GetNumberOfPages.vb
Imports System.IO
Imports Aspose.Note

Namespace Pages
    Public Class GetNumberOfPages
        Public Shared Sub Run()
            ' ExStart:GetNumberOfPages
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Pages()

            ' Load the document into Aspose.Note.
            Dim oneFile As New Document(dataDir & Convert.ToString("Aspose.one"))

            ' Get number of pages
            Dim count As Integer = oneFile.GetChildNodes(Of Page)().Count

            ' Print count on the output screen
            Console.WriteLine(count)
            ' ExEnd:GetNumberOfPages
        End Sub
    End Class
End Namespace
